tests/test_event_payload_descriptor.py

```diff
@@ -2,10 +2,10 @@
 
 import pytest
 
-from toadr3 import PayloadDescriptor, SchemaError
+from toadr3 import EventPayloadDescriptor, SchemaError
 
 
-def test_payload_descriptor():
```

Comment:
You currently do not test that the value EVENT_PAYLOAD_DESCRIPTOR is accepted for objectType.

Reply:
(Same for report payload descriptor tests)